python/dataproc_templates/gcs/README.md

```diff
@@ -0,0 +1,22 @@
+## GCS To BigQuery
+
+General Execution:
+
+```
+export GCP_PROJECT=<project_id> 
+export SUBNET=<region> 
```

Comment:
do we really need subnet here ?
Can we try and test without subnet ?

Reply:
I tested this without subnet. "gcloud beta dataproc batches submit pyspark" requires a subnet with private google access enabled. It uses default subnet if not explicitly provided. Our sandbox environment default subnet does not have private google access enabled.

Reply:
We can move the subnet to the optional options on the documentation. I'll be pushing some updates to the docs later, so I'll update it then.